File: src/main/kotlin/co/ao/nextbss/requirekt/ErrorWrapper.java
package co.ao.nextbss.requirekt;

import co.ao.nextbss.Yoru;
import java.util.List;

public class ErrorWrapper {
    private final List<ErrorResponse> errors;

    public ErrorWrapper(List<ErrorResponse> errors) {
        this.errors = errors;
    }

    public List<ErrorResponse> getErrors() {
        return errors;
    }

    public String toJsonString() {
        return new Yoru<ErrorWrapper>().toJson(this);
    }
}

File: src/main/kotlin/co/ao/nextbss/requirekt/ErrorWrapper.kt
package co.ao.nextbss.requirekt

import co.ao.nextbss.Yoru

open class ErrorWrapper {
    val errors = ArrayList<AbstractErrorResponse>()

    fun toJsonString(error: AbstractErrorResponse): String {
        errors.add(error)
        return Yoru<ErrorWrapper>().toJson(this)
    }
}